ray-operator/controllers/ray/raycluster_controller_fake_test.go

```diff
@@ -314,6 +314,11 @@ func setupTest(t *testing.T) {
 										},
 									},
 								},
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceMemory: resource.MustParse("100000000"),
+									},
+								},
```

Comment:
needed to make `common.ValidateHeadRayStartParams()` not return an error so we can move past it in `raycluster_controller.go` and trigger `utils.CheckAllPodsRunning()`.